activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -2426,6 +2426,9 @@ protected DestinationInterceptor[] createDefaultDestinationInterceptor() {
      * @throws IOException
      */
     protected Broker addInterceptors(Broker broker) throws Exception {
+        if (isAdvisorySupport()) {
```

Comment:
Please add a comment here that the order matters and include JIRA number

Reply:
done 👍